qldbdriver/result.go

```diff
@@ -19,8 +19,17 @@ import (
 	"github.com/aws/aws-sdk-go/service/qldbsession"
 )
 
-// Result is a cursor over a result set from a QLDB statement.
-type Result struct {
+// Result is an interface for a Result Set cursor.
+type Result interface {
```

Comment:
Please do not change this API.

Reply:
This interface is an addition to the code not a modification.
The method signatures in here are the methods shared between `result` and `BufferedResult` and also what is used by the tests dealing with a `Result` interface.

Reply:
I was referring to how Next(txn) was changed, but it's been reverted so this isnt an issue anymore.